build/media-suite_helper.sh

```diff
@@ -1451,7 +1451,7 @@ do_autoreconf() {
         -z "$(ls "$basedir"/build_successful* 2> /dev/null)" ]]; } ||
         [[ ! -f configure ]]; then
         extra_script pre autoreconf
-        log "autoreconf" autoreconf -fiv $*
+        log "autoreconf" autoreconf -fiv "$@"
```

Comment:
As long as this doesn't break with no parameters passed.

Reply:
As far as I saw, we have not used do_autoreconf with any params and the only one with params for autogen is fontconfig